Base/usr/share/man/man1/Applications/CrashReporter.md

```diff
@@ -0,0 +1,37 @@
+## Name
```

Comment:
In the commit message, typo:
`Importantly, this documents` => `Importantly, these documents`